frontend/src/components/boards/Board.vue

```diff
@@ -114,10 +139,10 @@ export default {
 			};
 		},
 		dragStart() {
-			console.log("drag started");
+			//console.log("drag started");
```

Comment:
Please remove unused codes and methods.

Reply:
It will be handy, during the implementation of "card" drag&drop